src/webui/api/torrentcreatorcontroller.cpp

```diff
@@ -112,6 +113,16 @@ TorrentCreatorController::TorrentCreatorController(BitTorrent::TorrentCreationMa
 {
 }
 
+QStringList parseUrls(const QString &urlsParam)
+{
+    const QStringList encodedUrls = urlsParam.split(u'|');
+    QStringList urls;
+    urls.reserve(encodedUrls.size());
+    for (const QString &urlStr : encodedUrls)
+        urls << QUrl::fromPercentEncoding(urlStr.toLatin1());
+    return urls;
+}
+
```

Comment:
Please move this function into the anonymous namespace at the top of this file. Specifically to line 91.